resources/fluid.py

```diff
@@ -7,16 +7,57 @@
 from scp.ethylene_glycol import EthyleneGlycol
 from scp.methyl_alcohol import MethylAlcohol
 from scp.propylene_glycol import PropyleneGlycol
-from scp.water import Water
+from scp.water import Water as ScpWater
```

Comment:
Any reason to call is `ScpWater`? Not that it really matters, but the rest aren't prepended with `Scp` even though they are from that library.